docs/contributing/common-scenarios.html

```diff
@@ -0,0 +1,94 @@
+{{>header}}
+
+{{>nav this}}
+
+	<div class="o-techdocs-main o-techdocs-content">
+
+		<h1>Common contribution scenarios</h1>
+
+		<p>A few scenarios come up time and again on our issue tracker. If you want to contribute, and one of the scenarios below descirbes what you want to change, follow the steps to acheive a painless and rapidly merged pull request!</p>
+
+
+		<h3 id='adding-browsers'>A particular polyfill is being served to the wrong browsers / not enough browsers</h3>
```

Comment:
The headings are long, and I didn't want them to appear in the left nav.  Fairly lousy reason but it does produce a better UX for most.